package.json

```diff
@@ -15,14 +15,14 @@
   "devDependencies": {
     "code": "5.x.x",
     "confidence": "3.x.x",
-    "dotenv": "5.x.x",
-    "eslint": "4.x.x",
-    "eslint-config-hapi": "11.x.x",
+    "dotenv": "6.x.x",
+    "eslint": "5.4.x",
```

Comment:
Hmm, I think if we list 5.x.x to reflect the compatibility that the boilerplate cares about and lab is more specific, then npm will install the more specific version anyway.

Reply:
👍 cool cool, i'll change that up then. thanks for reviewing! 🙏